rasa/core/training/structures.py

```diff
@@ -392,6 +399,10 @@ def dump_to_file(self, filename, flat=False, e2e=False):
 
 
 class StoryGraph(object):
+    """The graph of the story-steps pooled from all the stories
+    present in the training data.
```

Comment:
there should we one sentence description that fits in one line then blank line then additional description then `"""` in the new line then blank line

Reply:
Yeah, but I just have a one line description for now. So I just gotta move the triple quotes right after the period, right?

Reply:
but it doesn't fit on one line

Reply:
Does it have to? You're referring to something like this, right? https://github.com/RasaHQ/rasa/blob/f91a9caf6a04045fb6d86a4a04f4b2236f2df47b/rasa/core/featurizers.py#L23

Reply:
oh, this is wrong

Reply:
https://www.python.org/dev/peps/pep-0257/#multi-line-docstrings

Reply:
You're in for a refactoring treat, then 😄!

I came up with this regex to find such instances `:\n\h+""".*$\n\h+.*`, and I think there are around 286 matches.

Reply:
Oh wait, a lot of false positives.

Reply:
45 matches (`:\n\h+""".*$\n\h+.*`)

Reply:
What do you suggest we do?

Reply:
I fixed featurizer.py in my core_transformer PR.

Reply:
I've fixed mine. What about the other 43 instances, though?

Reply:
you can create separate PR 😂

Reply:
In my defence: "Some other good reasons to ignore a particular guideline:
- To be consistent with surrounding code that also breaks it"

(https://www.python.org/dev/peps/pep-0008/#a-foolish-consistency-is-the-hobgoblin-of-little-minds)

Reply:
ok, but I rather change the docstrings